src/Snackbar/Snackbar.types.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  SnackbarProps as MuiSnackbarProps,
+  SnackbarContentProps
+} from "@mui/material";
+
+export interface SnackbarProps {
+  /**
+   * Callback fired when the user clicks the action button.
+   *
+   * **Signature**
```

Comment:
I think we can remove signatures in the typescript because the storybook infers them for us.